Aptoide/appcoins-wallet-android
# [APPC-1443] - Integrated extractor.jar in OemIdExtractorService.kt.

**What does this PR do?**

Add support to v2 / v3 signed apks with injected oemid.
The injected string is a CSV where oemid is the first value.
extractor.jar is the library than handles the extraction properly.

**Database changed?**

   No

**Where should the reviewer start?**

- OemIdExtractorService.kt

**How should this be manually tested?**

You will need an APK with injected stuff in order to test the flow.
Complete an IAB flow with the injected app.
To verify it worked as expected, visit https://appcexplorer.io/ and confirm that your transaction has the correct OEM wallet address (the one mapped from the injected oemid).

**What are the relevant tickets?**

  Tickets related to this pull-request: https://aptoide.atlassian.net/browse/APPC-1443

**Code Review Checklist**
